gnosis/eth/tests/test_ethereum_client.py

```diff
@@ -765,7 +765,7 @@ def test_trace_filter(self):
             ValueError, "The method trace_filter does not exist/is not available"
         ):
             self.ethereum_client.parity.trace_filter(
```

Comment:
Maybe we should think in other name for `parity`, tracing sound like web3 methods what do you think?

Reply:
Sure, I fogot to rename it 😉